src/main/resources/localization/messages-nl.properties

```diff
@@ -0,0 +1,82 @@
+error.generic.ending = Er ging iets fout bij het beëindigen van deze giveaway.
+error.generic.delete = De giveaway is gestopt, maar er is iets foutgegaan tijdens het verwijderen van het giveaway-bericht.
+error.generic.creation = Er ging iets fout tijdens het maken van de giveaway.
+error.generic.enter = Er is iets fout gegaan tijdens het meedoen aan deze giveaway.
+error.generic.reroll = Er is iets fout gegaan tijdens het opnieuw kiezen van winnaars.
+error.generic = Er is iets fout gegaan.
+warning.no_giveaways = Momenteel zijn er geen giveaways bezig in deze server.
+warning.not_supported = Deze functie wordt momenteel niet ondersteund.
+warning.no_entries = Er zijn te weinig deelnemers, dus kon er geen winnaar worden gekozen!
+success.giveaway_created = De giveaway is succesvol gemaakt! ID: {0}
+success.entered = Je doet mee aan de giveaway!
+success.leave = Je doet niet meer mee aan de giveaway.
+success.ping = Pong!
+success.winner = Gefeliciteerd {0}! Je hebt **{1}** gewonnen!
+success.settings.role = Gebruikers met de {0} rol kunnen nu giveaways beheren!
+success.settings.color = De embed kleur voor giveaways is veranderd naar `{0}`.
+success.settings.emoji = De knop-emoji is veranderd naar `{0}`.
+success.giveaway_ended = De giveaway is succesvol beëindigd! {0}! 
+success.giveaway_reroll = {0} heeft de giveaway gererolld! Gefeliciteerd {1}!
+success.giveaway_delete = De giveaway {0} is succesvol verwijderd!
+info.invite = Hoi! Ik ben **GiveawayBot**! Ik help om giveaways snel en gemakkelijk te maken! \
+\nJe kan me toevoegen aan jouw server met deze link:
+info.about = Alles over {0}
+info.about.brief = Houd giveaways snel en gemakkelijk!
+info.about.long = Hoi! Ik ben **GiveawayBot**, Ik help om giveaways snel en gemakkelijk te laten verlopen in jouw \
+Discord server! Ik ben gemaakt door [**jagrosh**#4824](https://jagrosh.com/) (<@113156185389092864>). Al mijn commando''s \
+starten met `/g`. Vergeet niet om mijn website te bekijken! **https://giveawaybot.party/**!
+info.about.stats = Statistieken
+info.about.stats.servers = {0} servers
+info.about.stats.giveaways = {0} giveaways
+info.about.links = Links
+info.about.links.website = Website
+info.about.links.invite = Bot toevoegen
+info.about.links.support = Hulp
+info.settings = Instellingen
+info.settings.owner = Server eigenaar
+info.settings.premium = Premium Level
+info.settings.emoji = Giveaway Emoji
+info.settings.role = Manager Rol
+info.settings.color = Embed Kleur
+info.settings.locale = Taal
+info.settings.etc = Om te beheren welke rollen giveaways kunnen beheren, ga naar **Server Instellingen** > **Intergraties** > **GiveawayBot**
+info.help.general = Algemene Commando''s
+info.help.creation = Giveaway Aanmaak Commando''s
+info.help.manipulation = Giveaway Manipulatie Commando''s
```

Comment:
`Manipulatie` is mayhaps a bit too literally translated, and could be confusing in Dutch due to double meanings. Maybe `Bewerking` is better?

Reply:
Might be better indeed!